### PR TITLE
Revert useless commit for videos page

### PR DESCRIPTION
**Pull request by TheGamerzs**
image

### CODE REVIEW

File: websites/D/DEV Community/presence.ts [hunk 1]
```diff
@@ -161,10 +161,13 @@ presence.on("UpdateData", async () => {
...
-      if (contentStateKey === "play" && endTimestamp > 0)
+      if (contentStateKey == "play" && endTimestamp > 0) {
         presenceData.endTimestamp = endTimestamp;
-      else
+      } else {
+        delete presenceData.startTimestamp;
         delete presenceData.endTimestamp;
+      }
+      break;
       break;
```
Comment (callumok2004) [inline websites/D/DEV Community/presence.ts:171]: Why 2 breaks?

```
break;
break;
```

**Review by callumok2004 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by callumok2004 — DISMISSED** [review]
LGTM

**Review by Bas950 — APPROVED** [review]
LGTM

**Comment by ririxidev** [comment]
For what freaking purpose? XD

**Comment by ririxidev** [comment]
If it is useless for you, you didn't had to merge these things before merging your pull request. Now it stays like that.

**Pull request merged**